ocaml/xapi/cli_operations.ml

```diff
@@ -2665,16 +2665,22 @@ let get_snapshot_uuid params =
   then List.assoc "uuid" params
   else raise (failwith "Required parameter not found: snapshot-uuid or uuid.")
 
+let get_snapshotVM_by_uuid rpc session_id snap_uuid =
```

Comment:
Minor point: `get_snapshot_vm_by_uuid` would be more common. I don't request a change.